drivers/mtd_emulated/Kconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#               2020 HAW Hamburg
```

Comment:
Copyright needs update.

Reply:
I removed the file. It's not included anyway.